magenta/models/sketch_rnn/sketch_rnn_train.py

```diff
@@ -131,9 +131,9 @@ def load_dataset(data_dir, model_params, inference_mode=False):
     else:
       data_filepath = os.path.join(data_dir, dataset)
       if six.PY3:
```

Comment:
we now support only py3, so no need to have these two paths

Reply:
Ok! So is there a notification/message required to be given instead or should I remove it completely?

Marking @hardmaru too just in case.

Reply:
Done! :+1: